sql-mutations/make-yoder.sql

```diff
@@ -0,0 +1,7 @@
+update "actors"
+set "firstName" = 'Baby',
+     "lastName" = 'Yoda'
+where "actorId" = 15
```

Comment:
Does this work even though there is no `;` here?

Reply:
It works for me, but I will use ; next time